Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -157,6 +171,45 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
         this._fontVariationsGroup.rows = [...this._fontVariationRowsMap.values()];
     }
 
+    _createVariationInstanceRow(variationInstanceNames)
+    {
+        this._variationInstanceSelectElement = document.createElement("select");
+
+        this._variationInstanceDefaultOptionElement = this._variationInstanceSelectElement.appendChild(document.createElement("option"));
+        this._variationInstanceDefaultOptionElement.textContent = WI.UIString("None", "None @ Font Details Sidebar Instance", "Label to indicate that a font variation instance (preset) is not applied.");
+        this._variationInstanceDefaultOptionElement.disabled = true;
+        this._variationInstanceDefaultOptionElement.selected = this._matchingVariationInstanceName === null;
```

Comment:
It would be nice if the "None" option didn't appear when you currently match a known instance, since it isn't selectable.

Reply:
Unfortunately, it isn't possible to hide an `<option>` from a select input's dropdown with either CSS or the `hidden` attribute. The likely solution is to mutate the DOM to remove the `<option>` altogether. I hesitate to do that because as soon as a slider position changes to no longer match any named instances the `None` option should immediately appear and be selected in the dropdown. 

It would be nice, indeed, but I'll keep this as an enhancement for a follow-up.